python/ray/tune/hyperband.py

```diff
@@ -295,7 +297,9 @@ def current_trials(self):
 
     def continue_trial(self, trial):
         result = self._live_trials[trial]
-        if self._get_result_time(result) < self._cumul_r:
+        if self._get_result_time(result) > self._max_t:
```

Comment:
@richardliaw is this the right way to limit to max t? If not we can leave it for another pr.

Reply:
it's a little bit more involved than this..

Reply:
Ok, let me revert this then.